schema_utils.py

```diff
@@ -387,3 +388,86 @@ def is_valid_email(obj):
             bool. Whether the given object is a valid email.
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
+
+    @staticmethod
+    def is_valid_asciimath_expression(obj, algebraic=True):
```

Comment:
Name is now wrong, since you're not parsing against asciimath.

Reply:
Changed.